Client/Frontend/Intro/IntroViewController.swift

```diff
@@ -8,6 +8,8 @@ import SnapKit
 struct IntroViewControllerUX {
     static let Width = 375
     static let Height = 667
+    static let SyncButonTopPadding = 5
+    static let minimumFontScale = 0.5
```

Comment:
do we capitalize these (so far i just followed prevailing style in the existing file i was in)?